cookbooks/van-keken-vof/doc/van-keken-vof.tex

```diff
@@ -0,0 +1,216 @@
+Finally, since one is often interested in a high resolution image of the shape of the interface 
+between the two fluids, one can use the output of the VOF method to examine the interface 
+within individual cells or within regions consisting of groups of cells.
+
+Another advantage the VOF method has over modeling moving interface problems with compositional 
+fields is that for problems in which the interface occupies a relatively small part of the 
+computational domain, all of the computational work in the VOF method is done in cells that lie 
+in a neighborhood of the interface, rather than in the entire computational domain. 
+
+\begin{figure}[b!]
+   \centering
+\includegraphics[width=0.5\textwidth]{cookbooks/van-keken-vof/doc/rms_vel_comparison.png}
+   \caption{Evolution of the root mean square velocity as a function of time for computations 
+    of the Van Keken problem made with the VOF interface tracking algorithm with five 
+    different global mesh refinements.
+    Since the VOF initial conditions are discontinuous, the above results should really be 
+    compared with the computations with discontinuous initial conditions on the left in 
+    Figure~\ref{fig:vk-2}. However, the above results also compare extremely favorably with the 
+    computations with smoothed, continuous initial conditions for the compositional field on 
+    the right in Figure~\ref{fig:vk-2}.
+    As in Figure~\ref{fig:vk-2}, 5 global refinements correspond to a $32 \times 32$ mesh and 10 global refinements correspond to a $1024 \times 1024$ mesh.}
+    \label{fig:vof-vk-1}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.4\textwidth]{cookbooks/van-keken-vof/doc/vof_van_keken_refinement_comparison.png}
+    \caption{\it The results of two computations of the Van Keken problem made with the VOF 
+        interface tracking algorithm overlaid upon each other at $t_{end}=2000$.
+        This visualization shows the reconstructed boundary between the two
+        materials at the final time $t_{end}$ as computed on a uniform grid
+        with 7 and 8 levels of refinement.
+        The boundaries between the materials are displayed as contours
+        of the fields $\tilde{\psi}^7\,(t_{end})$ (black) and
+        $\tilde{\psi}^8\,(t_{end})$ (bright green), which are generated by the  visualization postprocessor.
+        The contours for the reconstructed material boundaries are superimposed
+        on a color gradient visualization of the material composition for the
+        computation with 8 levels of refinement in order to make the regions
+        with each fluid type more evident.
+        Compare with the fourth image on the right in Figure~\ref{fig:vk-1}.
+    }
+    \label{fig:VOF_van_Keken-02}
+\end{figure}
+
+Since, as noted above, when the interface is discontinuous, the van Keken problem is a version 
+of the Rayleigh-Taylor problem that is unstable to perturbations of all 
```

Comment:
```suggestion
of the Rayleigh-Taylor problem, which is unstable to perturbations of all 
```
I believe this is a case where the rules of grammar specify that one should use the word 'which' rather than 'that'.

The comma preceding the word 'which' is part of Cambridge University Press' "Guide for Authors".

Reply:
Fixed and commited.